src/content/docs/more-integrations/open-source-telemetry-integrations/airflow/monitoring-airflow-ot.mdx

```diff
@@ -0,0 +1,142 @@
+
+<img
+  title="Screenshot showing sample Airflow DAG runs in New Relic"
+  alt="Screenshot showing sample Airflow DAG runs in New Relic"
+  src={opentelemetryAirflow01}
+/>
+
+## Prerequisites [#prerequisites]
+
+You need to first install the Airflow package with OpenTelemetry using the `otel` extra. This can be done in one of several ways, depending on how you deploy Airflow:
+
+### Installing from PyPi [#install-pypi]
+
+1. Follow the installation instructions from [Airflow's Documentation](https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html).
+2. When installing with pip, add the `otel` extra to the command.
+  a. eg. `pip install "apache-airflow[otel]"`
+
+### Installing from Docker [#install-docker]
+
+1. Setup the [Airflow Docker image](https://airflow.apache.org/docs/docker-stack/index.html) using instructions from [Airflow's Documentation](https://airflow.apache.org/docs/docker-stack/index.html).
+2. Extend the prebuilt image by using a Dockerfile to install the `otel` extra. You can replace the latest tag with your desired version of the Airflow image.
+
+```
+FROM apache/airflow:latest
+RUN pip install --no-cache-dir "apache-airflow[otel]==${AIRFLOW_VERSION}"
+```
+
+## Configuration [#configuration]
+
+You need to configure the Airflow OpenTelemetry metrics to send data to an [OpenTelemetry Collector](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-intro/) that reports data to a New Relic [OTLP Endpoint](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup/#note-endpoints) using a <InlinePopover type="licenseKey" />.
+
+<Callout variant="important">
+  Airflow does not currently supporting sending OpenTelemetry data with authentication headers, making the OpenTelemetry Collector a requirement to authenticate with New Relic.
+</Callout>
+
+### OpenTelemetry Collector Configuration [#configuration-collector]
+
+1. Follow the [basic collector example](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-basic/) to setup your OpenTelemetry Collector.
+2. Configure it with your appropriate OTLP Endpoint. For example, `https://otlp.nr-data.net:4317`.
+3. For authentication, add your <InlinePopover type="licenseKey" /> to the environment variable `NEW_RELIC_LICENSE_KEY` so that it populates the `api-key` header.
+4. Ensure port 4318 on the collector is open and available to the running Airflow instance. (For docker, you may need to use a [docker network](https://docs.docker.com/network/).)
+5. Launch your collector.
+
+### Airflow Metrics Configuration [#configuration-airflow]
+
+Airflow sends metrics using OTLP over HTTP. To send this data to our OpenTelemetry Collector it will need to be sent to port 4318. Airflow has multiple methods of [setting configuration options](https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html).
```

Comment:
```suggestion
Airflow sends metrics using OTLP over HTTP. To send this data to the OpenTelemetry collector, you'll need to use port `4318`. Airflow has multiple methods of [setting configuration options](https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html).
```

Reply:
Actually, can we remove this? It's implied in the previous step that they already set the port to 4318.

Reply:
I wanted to make this crystal clear because the docs for Airflow and Airflow's default setting are straight up wrong and say to use port 8889 which is likely copy paste error from prometheus metrics.

I was planning to PR that change but I'm not sure if I feel like navigating multiple repos and Apache's review process. We'll see if I get around to it.

Reply:
I reformatted this to be shorted, see if that's better.